docs/dev/MAINTAINERS.md

```diff
@@ -0,0 +1,142 @@
+# **Coffee**: Hacking guide
+
+## Table of Content
```

Comment:
this need to be inside the https://github.com/coffee-tools/coffee/blob/master/docs/docs-book/src/how-to-contribute.md I think is just a copy and paste?